docs/usage/llms/local-llms.mdx

```diff
@@ -6,57 +6,69 @@ description: When using a Local LLM, OpenHands may have limited functionality. I
+### 1. Install LM Studio
 
-3. Open LM Studio application, you should first switch to `power user` mode, and then open the developer tab:
+Download and install the LM Studio desktop app from [lmstudio.ai](https://lmstudio.ai/).
 
-![image](./screenshots/1_select_power_user.png)
+### 2. Download Devstral Small
 
-4. Then click `Select a model to load` on top of the application:
+1. Make sure to set the User Interface Complexity Level to "Power User", by clicking on the appropriate label at the bottom of the window.
+2. Click the "Discover" button (Magnifying Glass icon) on the left navigation bar to open the Models download page.
 
-![image](./screenshots/2_select_model.png)
+![image](./screenshots/01_lm_studio_open_model_hub.png)
 
-5. And choose the model you want to use, holding `option` on mac to enable advanced loading options:
+3. Search for the "Devstral Small 2505" model, confirm it's the official Mistral AI (mistralai) model, then proceed to download.
 
-![image](./screenshots/3_select_devstral.png)
+![image](./screenshots/02_lm_studio_download_devstral.png)
 
-6. You should then pick an appropriate context window for OpenHands based on your hardware configuration (larger than 32768 is recommended for using OpenHands, but too large may cause you to run out of memory); Flash attention is also recommended if it works on your machine.
+4. Wait for the download to finish.
 
-![image](./screenshots/4_set_context_window.png)
+### 3. Load the Model
 
-7. And you should start the server (if it is not already in `Running` status), un-toggle `Serve on Local Network` and remember the port number of the LMStudio URL (`1234` is the port number for `http://127.0.0.1:1234` in this example):
+1. Click the "Developer" button (Console icon) on the left navigation bar to open the Developer Console.
+2. Click the "Select a model to load" dropdown at the top of the application window.
 
-![image](./screenshots/5_copy_url.png)
+![image](./screenshots/03_lm_studio_open_load_model.png)
 
-8. Finally, you can click the `copy` button near model name to copy the model name (`imported-models/uncategorized/devstralq4_k_m.gguf` in this example):
+3. Enable the "Manually choose model load parameters" switch.
+4. Select 'Devstral Small 2505' from the model list.
 
-![image](./screenshots/6_copy_to_get_model_name.png)
+![image](./screenshots/04_lm_studio_setup_devstral_part_1.png)
 
-### Start OpenHands with locally served model
+5. Enable the "Show advanced settings" switch at the bottom of the Model settings flyout to show all the available settings.
+6. Set "Context Length" to at least 32768 and enable Flash Attention.
```

Comment:
nit, number could probably use comma and units, like `32,768 MB` (not sure if the ui says mib or what)